metron-deployment/inventory/quick-dev-platform/group_vars/all

```diff
@@ -0,0 +1,95 @@
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements.  See the NOTICE file distributed with
+#  this work for additional information regarding copyright ownership.
+#  The ASF licenses this file to You under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with
+#  the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+# which services should be started?
+services_to_start:
+  - mysql
+  - elasticsearch
+  - pcap-service
+  - kibana
+  - snort
+  - snort-logs
+  - bro
+  - pcap-replay
+  - bro-parser
+  - snort-parser
+  - enrichment
+  - indexing
```

Comment:
Yes! We finally split the config for quick-dev from full-dev. What if we didn't start anything by default for quick-dev?  When you're developing with quick-dev, you just go in and manually start just what you need.